### ISSUE TITLE
specifying a connector name

### ISSUE THREAD

**Issue by thebelovedmoon** [silexlabs/Silex#1179]
hello! I'm new to Silex as I'm migrating websites to my GitHub repository. I'm test publishing a page to see if it works correctly when I got this error.

![image](https://user-images.githubusercontent.com/103877195/163707015-60603789-dd90-4dd0-b2d9-050eecd52612.png)

in this instance, **publishing a website requires a connector name**, which I don't know what it is and where it's from. (and it kinda doesn't apply in the documentation which is ages old, I'm assuming)

any help is appreciated^^

**Comment by lexoyo** [comment]
Hello
You are using editor.silex.me?
What did you choose in the publication screen before this one?

**Comment by thebelovedmoon** [comment]
yes I'm using web editor (hence editing.silex.me, in some cases staging.silex.me)

first thing I did was authenticate through GitHub (this one) by publishing it as a test, then followed the procedures (adding and configuring repositories), then that's basically it

**Comment by lexoyo** [comment]
Can you screenshot the publish settings?

Also try clicking "edit" instead of "continue" after you clicked "publish"

**Comment by thebelovedmoon** [comment]
aight I'll send over some later^^

**Comment by lexoyo** [comment]
Thank you for reporting this and for trying out Silex!

This was a v2 issue related to the old publishing system. Silex v3 has a completely new connector architecture that works differently.

We're closing this as part of a tracker cleanup. If you'd like to try v3, head over to https://v3.silex.me — and don't hesitate to open a new issue if you run into any trouble!
